### PR TITLE
[mkbundle] Automatically bundle config files and machine.config files

### PR DESCRIPTION
**Pull request by migueldeicaza**
For the simple/cross compiler scenario, fixes one of the missing issues for #6230

### CODE REVIEW

File: mcs/tools/mkbundle/mkbundle.cs [hunk 1]
```diff
@@ -514,6 +531,7 @@ static int Main (string [] args)
 
 	static void VerifySdk (string path)
 	{
+		Console.WriteLine ("at: " + path);
```
Comment (marek-safar) [inline mcs/tools/mkbundle/mkbundle.cs]: Debugging leftover

File: mcs/tools/mkbundle/mkbundle.cs [hunk 2]
```diff
@@ -289,6 +300,12 @@ static int Main (string [] args)
 				if (!quiet)
 					Console.WriteLine ("WARNING:\n  Check that the machine.config file you are bundling\n  doesn't contain sensitive information specific to this machine.");
 				break;
+                        case "--no-machine-config":
```
Comment (marek-safar) [inline mcs/tools/mkbundle/mkbundle.cs]: nit: indentation

Comment (migueldeicaza) [reply mcs/tools/mkbundle/mkbundle.cs]: No clue what you mean.

Comment (akoeplinger) [reply mcs/tools/mkbundle/mkbundle.cs]: @migueldeicaza you're using spaces instead of tabs here

Comment (migueldeicaza) [reply mcs/tools/mkbundle/mkbundle.cs]: Got it - done

**Pull request merged**